katsdptelstate/aio/test/test_telescope_state.py

```diff
@@ -31,520 +33,512 @@
 from katsdptelstate.aio.redis import RedisBackend, _DUMMY_CHANNEL
 
 
-class TestTelescopeState(asynctest.TestCase):
-    async def setUp(self) -> None:
-        self.ts = await self.make_telescope_state()
-        self.ns = self.ts.view('ns')
+class TestTelescopeState:
```

Comment:
What is the benefit of keeping the class as opposed to a bunch of fixtures and functions? I'm currently weighing this up in my own code...

Reply:
Mostly just minimising the amount of code I needed to change (my goal to limit the time I spent on this PR). In some cases I've kept classes around just to logically group the tests, or because subclasses perform a sort of parametrization by overriding certain behaviour. Not sure offhand if either is applicable here.

Reply:
Yes, in this case it's using poor-man's parametrization: there are classes like TestTelescopeStateRedis that subclass this class and override `make_telescope_state`.